lib/rules/function-no-unknown/__tests__/index.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const { messages, ruleName } = require('..');
+
+testRule({
+	ruleName,
+	config: true,
+	skipBasicChecks: true,
+
+	accept: [
+		{
+			code: 'a { transform: translate(1px); }',
+		},
+		{
+			code: 'a { transform: TRANSLATE(1px); }',
+		},
+		{
+			code: 'a { transform: --custom-function(1px); }',
+		},
+		{
+			code: 'a { transform: scale(0.5) translate(-100%, -100%); }',
+		},
+	],
+
+	reject: [
+		{
+			code: 'a { transform: unknown(4); }',
+			message: messages.rejected('unknown'),
+			line: 1,
+			column: 16,
+		},
+		{
+			code: 'a { transform: UNKNOWN(4); }',
+			message: messages.rejected('UNKNOWN'),
+			line: 1,
+			column: 16,
+		},
+		{
+			code: 'a { width: calc(10% * unknown(1)); }',
+			message: messages.rejected('unknown'),
+			line: 1,
+			column: 23,
+		},
+	],
+});
+
+testRule({
+	ruleName,
+	config: true,
+	skipBasicChecks: true,
```

Comment:
```suggestion
	ruleName,
	config: true,
	customSyntax: "postcss-scss",
	skipBasicChecks: true,
```

Let's specify the SCSS syntax (even though the default parser can parse the construct).